packages/documentation/docs/controls/_forms_custom_field_styleguide.md

```diff
@@ -0,0 +1,43 @@
+The custom field is a wrapper component that can host any other forms component. It provides a set of properties that allow you to control the validation state of the field, as well as the helper text that is displayed below the field. The custom field is a versatile tool to create your own form fields, that can be used in combination with the 'Form' components to create complex forms.
```

Comment:
```suggestion
The custom field is a wrapper component that can host any forms component. Its properties allows you to control the validation state of the field and the helper text. The custom field is a versatile tool to create your own form fields, that can be used in combination with the 'Form' components to create complex forms.
```

I try to shorten the first part a bit. For me, the second part sounds kinda repetitive, though I like it ... 🤔